src/run_build.py

```diff
@@ -70,10 +71,15 @@ def main() -> int:
                 logging.info(f"Successfully built {component.name}")
             except:
                 logging.error(f"Error building {component.name}, retry with: {args.component_command(component.name)}")
-                raise
+                if args.continue_on_error and component.name not in ['OpenSearch', 'OpenSearch Dashboards']:
```

Comment:
Is the component name `OpenSearch Dashboards` or OpenSearch-Dashboards`?

Reply:
fixed it to `OpenSearch-Dashboards`.